src/Tasks.UnitTests/Copy_Tests.cs

```diff
@@ -2075,10 +2095,10 @@ public void CopyWithHardAndSymbolicLinks()
 
                 bool success = t.Execute();
 
-                Assert.False(success);
-
+                Assert.True(success);
+                me.AssertLogContains("0x80070011");
```

Comment:
When verify the logs,  for some reason when this test runs with all other tests we are getting the incorrect result from CreateHardLink error message (a message associated with another test). Calling GetHRForLastWin32Error 
 GetExceptionForHR seems to clear out the previous message and allow us to get the right message in the Copy task.  But for mac and linux. the error is still there.
https://dev.azure.com/dnceng-public/public/_build/results?buildId=101777&view=ms.vss-test-web.build-test-results-tab&runId=2180158&resultId=104184&paneView=debug